rosidl_generator_java/resource/srv.cpp.em

```diff
@@ -1,23 +1,65 @@
 @# Included from rosidl_generator_java/resource/idl.cpp.em
 @{
+import os
+from rosidl_cmake import expand_template
 from rosidl_generator_c import idl_structure_type_to_c_include_prefix
 
-service_includes = [
+namespaces = service.namespaced_type.namespaces
+type_name = service.namespaced_type.name
+request_type_name = service.request_message.structure.namespaced_type.name
+response_type_name = service.response_message.structure.namespaced_type.name
+
+data = {
+    'package_name': package_name,
+    'output_dir': output_dir,
+    'template_basepath': template_basepath,
+}
+
+# Generate request message
+data.update({'message': service.request_message})
+output_file = os.path.join(
+    output_dir, *namespaces[1:], '{0}.ep.{1}.cpp'.format(request_type_name, typesupport_impl))
+expand_template(
+    'msg.cpp.em',
+    data,
+    output_file)
+
+# Generate response message
+data.update({'message': service.response_message})
+output_file = os.path.join(
+    output_dir, *namespaces[1:], '{0}.ep.{1}.cpp'.format(response_type_name, typesupport_impl))
+expand_template(
+    'msg.cpp.em',
+    data,
+    output_file)
+
+jni_includes = [
```

Comment:
This looks a bit overly complicated, why not just add the includes right in the template instead of in variables?

Reply:
Agreed. This was left-over from a previous iteration. Simplified in dc5d64d